src/InteractiveSlide/InstantBook.php

```diff
@@ -273,7 +290,7 @@ function (CacheItemInterface $item) use ($now): \DateTime {
         );
 
         if ($lastRequestDateTime->add(new \DateInterval(self::CACHE_LIFETIME_QUICK_BOOK_SPAM_PROTECT)) > $now) {
-            throw new ServiceUnavailableHttpException(60);
+            throw new InteractiveSlideException('Service unavailable', 503);
```

Comment:
```suggestion
            throw new InteractiveSlideRateLimitException('To many requests');
```
Then create that exception and map it in config to [429 Too Many Requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/429)

Note: Symfony has a rate limiting component, ideally request that exceed a rate limit should never reach the domain layer but be handled in the controller layer